### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by MohammadReza0GH** [phoebe-project/phoebe2#866] [Questions]
how ca i fix this error ? : 
2G model C failed with error: Residuals are not finite in the initial point.
2G model CE failed with error: Residuals are not finite in the initial point.
2G model CG failed with error: Residuals are not finite in the initial point.
2G model CGE failed with error: Residuals are not finite in the initial point.
2G model CG12 failed with error: Residuals are not finite in the initial point.
2G model CG12E1 failed with error: Residuals are not finite in the initial point.
2G model CG12E2 failed with error: Residuals are not finite in the initial point.

TypeError: ce() missing 2 required positional arguments: 'Aell' and 'phi0'

**Comment by MohammadReza0GH** [comment]
this happens in lc_geometry run solver.

**Reply by kecnry** [reply]
This is a known issue - see #849 - and can usually be avoided by modifying or disabling the phase binning options. Hopefully we can get to a proper "fix" sometime soon.
